src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -0,0 +1,206 @@
+import { getFormattedDate } from '../../../app/case/answers/formatDate';
+import { getAnswerRows } from '../../../app/case/answers/getAnswerRows';
+import { Checkbox } from '../../../app/case/case';
+import { YesOrNo } from '../../../app/case/definition';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { Sections } from '../../applicant1Sequence';
+import { CommonContent } from '../../common/common.content';
+import * as urls from '../../urls';
+
+const en = ({ isDivorce, partner, formState }: CommonContent) => ({
```

Comment:
is alot of these duplicates from app1 and 2 CYA? if so maybe we can import it from them and add extra stuff we need.

Reply:
There's only a small overlap - 1 question and 3 answers, I'm not sure it would be worth it